index.html

```diff
@@ -6,19 +6,40 @@
     <meta name="viewport" content="width=device-width, initial-scale=1.0" />
     <title>portofolio: setup and mobile version skeleton</title>
     <link rel="stylesheet" href="style.css" />
+    <script defer src="menu.js"></script>
+    <link
+      rel="stylesheet"
+      href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/6.1.1/css/all.min.css"
+      integrity="sha512-KfkfwYDsLkIlwQp6LFnl8zNdLGxu9YAA1QvwINks4PhcElQSvqcyVLLD9aMhXd13uQjoXtEKNosOWaZqXgel0g=="
+      crossorigin="anonymous"
+      referrerpolicy="no-referrer"
+    />
   </head>
-  <body class="">
+  <body class=" scrollable ">
     <header class="navibar">
       <nav class="nav-container">
-        <a class="logo" href="#">My Logo</a>
-        <img class="mobile-menu" src="img/Union.png" alt=" navigation" />
+        <a id="logo" class="logo" href="#">My Logo</a>
+
         <ul class="nav-list">
-          <li class="navi-item"><a href="#portofolio">portofolio</a></li>
-          <li class="navi-item"><a href="#about">About</a></li>
+          <li class="navi-item"><a href="#Portofolio">Portofolio</a></li>
+          <li class="navi-item"><a href="#About">About</a></li>
```

Comment:
- [ ] Great job so far, when I click the portfolio and about the button, it doesn't lead to the respective part of the page, please fix that. _Hint check for case sensitivity_.